apputils/config_validator.py

```diff
@@ -0,0 +1,52 @@
+"""Validator for config files."""
+
+
+class ConfigValidator(object):
```

Comment:
Should have tests for all functions in here. For configparser stuff, use its native exceptions for hte assertions

Reply:
Uses configparser native functionality